### PR TITLE
Feature/renderer refactor

### PR DESCRIPTION
**Pull request by inyono**
@Knorrke need a second pair of eyes here 👀

### CODE REVIEW

File: packages/server/src/module/Renderer/src/Renderer/Factory/RendererStorageFactory.php [hunk 1]
```diff
@@ -51,9 +51,6 @@ public function createService(ServiceLocatorInterface $serviceLocator)
                 'serializer',
             ],
         ];
-        if (array_key_exists('markdown_cache', $serviceLocator->get('Config'))) {
```
Comment (Knorrke) [inline packages/server/src/module/Renderer/src/Renderer/Factory/RendererStorageFactory.php]: did you check if theres a production config for this?

File: packages/server/src/module/Ui/templates/entity/view/default.twig [hunk 2]
```diff
@@ -20,7 +20,7 @@
  # @link      https://github.com/serlo-org/serlo.org for the canonical source repository
  #}
 {% set convert = convert is defined and convert %}
-{% set renderer = convert or entity().isOryEditorFormat(entity.getCurrentRevision()) ? oryRenderer() : markdown() %}
+{% set renderer = convert or entity().isLegacyFormat(entity.getCurrentRevision()) ? editorRenderer() : legacyEditorRenderer %}
```
Comment (Knorrke) [inline packages/server/src/module/Ui/templates/entity/view/default.twig]: not isLegacyFormat

File: packages/server/src/module/Page/src/Page/Entity/PageRevision.php [hunk 3]
```diff
@@ -147,7 +147,7 @@ private function injectFromArray($key, array $array, $default = null)
         }
     }
 
-    public function isOryEditorFormat()
+    public function isLegacyFormat()
```
Comment (Knorrke) [inline packages/server/src/module/Page/src/Page/Entity/PageRevision.php]: fixme. Probably replace calls to this with LegacyFormatHelper

File: packages/server/src/module/Entity/src/Entity/View/Helper/EntityHelper.php [hunk 4]
```diff
@@ -89,7 +89,7 @@ public function getOptions(EntityInterface $entity)
         );
     }
 
-    public function isOryEditorFormat(RevisionInterface $revision)
+    public function isLegacyFormat(RevisionInterface $revision)
```
Comment (Knorrke) [inline packages/server/src/module/Entity/src/Entity/View/Helper/EntityHelper.php]: fixme. Probably replace calls to this with LegacyFormatHelper

File: packages/server/src/module/Renderer/src/Renderer/Factory/RendererStorageFactory.php (hunk 1 above)
Comment (inyono) [reply packages/server/src/module/Renderer/src/Renderer/Factory/RendererStorageFactory.php]: yep

**Comment by inyono** [comment]
This is fully merged into #27, right? So we can close this I guess

**Pull request closed without merging**